redbot/cogs/downloader/downloader.py

```diff
@@ -36,6 +36,17 @@
 
 @cog_i18n(_)
 class Downloader(commands.Cog):
+    """Install community cogs made by Cog Creators.
+
+    Community cogs, also called third party cogs, are not included
+    in the default Red install.
+    
+    Community cogs always come in repositories. Repos are a
+    group of cogs by one creator you can install. You always need
```

Comment:
I don't like the word "always" here, mostly because if you already installed the repo before, if you just want another cog you don't have to add the repo again.